apps/namegraph.dev/lib/utils.ts

```diff
@@ -1,6 +1,126 @@
 import { clsx, type ClassValue } from "clsx";
 import { twMerge } from "tailwind-merge";
+import {
+  NameGraphGroupedByCategoryResponse,
+  NameGraphGroupingCategory,
+  NameGraphSuggestion,
+} from "@namehash/namegraph-sdk/utils";
+import { createNameGraphClient } from "@namehash/namegraph-sdk";
 
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+interface LabelAndModeQueryParams {
+  label: string;
+  mode: "full";
+}
+
+export interface NameGraphRelatedCollection {
+  collection_id: string;
+  collection_title: string;
+  collection_members_count: number;
+}
+
+export interface NameGraphSuggestionCategory {
+  type: NameGraphGroupingCategory;
+  // NameGraph API guarantees "name" will always be a normalized name
+  name: string;
+  // NameGraph API guarantees "suggestions" will never be an empty list
+  suggestions: NameGraphSuggestion[];
+  related_collections: NameGraphRelatedCollection[];
+}
+
+// This array is used for checking when a category is not a "related" category
+const NameGraphSuggestionCategoryTypes = Object.values(
```

Comment:
Seems this idea should be moved into the SDK?

Reply:
As this is a representation of the same NameGraph SDK present utility, but, with a different data structure, this is very much related to clients. I say this because apps might want to use this is different ways and mapping from one data structure to another one may be a responsibility of the client side, I think, so I wouldn't migrate this, what do you think?